test/codeu/chat/util/TokenizerTest.java

```diff
@@ -0,0 +1,57 @@
+package codeu.chat.util;
+import java.util.*;
+import codeu.chat.util.Tokenizer;
+import org.junit.Test;
+import java.io.IOException;
+import static org.junit.Assert.*;
+public final class TokenizerTest {
+	@Test
+	public void testWithNoQuotes() throws IOException {
+		final Tokenizer tokenizer = new Tokenizer("hello world how are you");
+		while (tokenizer.hasNext()){
```

Comment:
Tokenizer doesn't have a hasNext function, which leads to this line causing an error when building.